Formula/acl2.rb

```diff
@@ -4,45 +4,46 @@ class Acl2 < Formula
   url "https://github.com/acl2/acl2/archive/8.3.tar.gz"
   sha256 "45eedddb36b2eff889f0dba2b96fc7a9b1cf23992fcfdf909bc179f116f2c5ea"
   license "BSD-3-Clause"
-  revision 4
+  revision 5
 
   bottle do
     sha256 "47d523299e219e13e26adb3e3cc4d2eb984e9d535ef242bf16ee4c92229b63aa" => :big_sur
     sha256 "8bfb7ce324dfa93e6d8f46acaafbf69be4ed400c101371af97aa6ff7d9503469" => :catalina
     sha256 "547ba61b3a0514bd55ff3a04b19c4e5792ec19eb64f46cbb672bf30ffd871b93" => :mojave
   end
 
-  depends_on "sbcl"
+  depends_on "clozure-cl"
+  depends_on "openssl@1.1"
+  depends_on "z3"
 
   def install
     system "make",
-           "LISP=#{HOMEBREW_PREFIX}/bin/sbcl",
-           "ACL2=#{buildpath}/saved_acl2",
-           "USE_QUICKLISP=0",
-           "all", "basic"
-    system "make",
-           "LISP=#{HOMEBREW_PREFIX}/bin/sbcl",
-           "ACL2_PAR=p",
-           "ACL2=#{buildpath}/saved_acl2p",
-           "USE_QUICKLISP=0",
-           "all", "basic"
+      "LISP=#{Formula["clozure-cl"].opt_bin}/ccl64",
+      "ACL2_PAR=p",
+      "ACL2_REAL=r",
+      "ACL2=#{buildpath}/saved_acl2pr",
+      "USE_QUICKLISP=1",
+      "all", "basic"
     libexec.install Dir["*"]
 
     (bin/"acl2").write <<~EOF
       #!/bin/sh
-      export ACL2_SYSTEM_BOOKS='#{libexec}/books'
-      #{Formula["sbcl"].opt_bin}/sbcl --core '#{libexec}/saved_acl2.core' --userinit /dev/null --eval '(acl2::sbcl-restart)'
-    EOF
-    (bin/"acl2p").write <<~EOF
-      #!/bin/sh
-      export ACL2_SYSTEM_BOOKS='#{libexec}/books'
-      #{Formula["sbcl"].opt_bin}/sbcl --core '#{libexec}/saved_acl2p.core' --userinit /dev/null --eval '(acl2::sbcl-restart)'
+      IMAGE=`find #{libexec} -name 'saved_acl2pr.*' | head -n1`
+      if [ -z "$IMAGE" ]
+      then
+        echo "ACL2 image is not found in #{libexec}" 1>&2
+        exit 1
+      else
```

Comment:
This should really be up to upstream to define and maintain

Reply:
I agree with you. I will transfer this issue and ask improvement of installation to them.
I remove launcher script. As you said, we should not use wrapper script for future updates.

Next, I'd like to discuss the solution to run bottle made from current formulae.